dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/ProgramControllerIntegrationTest.java

```diff
@@ -88,76 +76,6 @@ public void testSetup() throws JsonProcessingException {
     POST("/metadata", WebClient.Body("program/create_program.json")).content(HttpStatus.OK);
   }
 
-  @Test
```

Comment:
instead of removing the test, could it be updated so it still provides value and confirms at least one of the expected behaviours? (either copying enrollments or not copying enrollments)

Reply:
sure, I can test tracker program enrollments are not copied.
Unfortunately I can't test event program enrollments are because they are not returned in the api response.

Reply:
I'll add this test in master as well.

Reply:
nice one 👍